packages/nx-plugin-openapi/src/test/mockSpawn.ts

```diff
@@ -6,21 +6,23 @@ export function mockSpawn(
 ) {
   const mock = spawn as jest.Mock;
   for (const invocation of invocations) {
-    mock.mockImplementationOnce((command: string, args: string[]) => {
+    mock.mockImplementationOnce((command: string, args: string[], options: { stdio: 'ignore' | 'pipe' }) => {
       expect([command, ...args]).toEqual([invocation.command, ...invocation.args]);
 
       const child: any = new EventEmitter();
       child.stdin = new EventEmitter();
       child.stdin.write = jest.fn();
       child.stdin.end = jest.fn();
-      child.stdout = new EventEmitter();
-      child.stderr = new EventEmitter();
+      if (options.stdio !== 'ignore') {
```

Comment:
This is becoming uncomfortably close to mocking the node spawn implementation details in my opinion

Reply:
Right? At the time it seemed the only way for testing the executor. Do you have anything in mind?